queries/ruby/highlights.scm

```diff
@@ -38,6 +46,11 @@
 (method
   "end" @keyword.function)
 
+(call
+  method: (identifier) @keyword.function
+  (#any-of? @keyword.function
+    "alias_method" "define_method" "define_singleton_method" "remove_method" "undef_method"))
```

Comment:
Also need to be moved below- and seems like these should also be `@function.builtin`?